config/samples/resources/accesscontextmanagerserviceperimeterresource/accesscontextmanager_v1beta1_accesscontextmanagerserviceperimeterresource.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1beta1
+kind: AccessContextManagerServicePerimeterResource
+metadata:
+  name: accesscontextmanagerserviceperimeterresourcesample
+spec:
+  perimeterNameRef:
+    # Referencing a resource name leads to recursive reference and KCC does not support the feature for now.
```

Comment:
OK ... let's file a github issue so we can follow up on this

Reply:
Created created https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/1309 and added TODO